config/initializers/admin_panel.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.config.after_initialize do
   if Spree::Core::Engine.backend_available?
-    Rails.application.config.spree_backend.main_menu.insert_before('users', ::SpreeProductQuestion::Admin::MainMenu::QuestionBuilder.new.build)
+    Rails.application.config.spree_backend.main_menu.insert_before('users', SpreeProductQuestion::Admin::MainMenu::QuestionBuilder.new.build)
+
+    Rails.application.config.spree_backend.tabs[:product].add(SpreeProductQuestion::Admin::Tabs::ProductTabsBuilder.new.build)
   end
-end
\ No newline at end of file
+end
```

Comment:
some of your files don't have new line at end of file and some have, 
could you please stick to the patterns of this repository and be consistent about this